pkgs/applications/misc/command-injection-tester/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "command-injection-tester";
+  version = "dbcf85f7fdb173028a8af461e2c0f8bfcac42b73";
```

Comment:
```suggestion
  version = "0.0.2-unstable-2024-04-28";
```

Reply:
Not sure how I hadn't spotted this pattern in other packages previously, but I'll do this too, thanks! :)